content/docs/intro/vs/cloud_templates.md

```diff
@@ -11,21 +10,25 @@ menu:
 aliases: ["/docs/reference/vs/cloud_templates/"]
 ---
 
+## What is CloudFormation?
+
 All major cloud providers offer their own form of infrastructure-as-"code" solution, typically by way of JSON or
 YAML-based templating solutions. This includes AWS CloudFormation and Azure Resource Manager (ARM) templates.
 
 These markup-based configuration files are often uploaded to a hosted service in the target cloud, where a hosted
 service will then process the files to create, update, or delete resources as necessary.
 
-Pulumi's model shares a lot with these systems. Although Pulumi programs are written in imperative, familiar languages,
-they are ultimately evaluated to produce a similar set of create, update, or delete operations for your cloud resources.
+## The Pulumi Difference
 
-This is where Pulumi begins to diverge from these other solutions, however.
+Pulumi offers a multi-cloud alternative to more restrictive infrastructure-as-code template solutions, such as AWS CloudFormation. Pulumi allows you to streamline processes through managing resources from different platforms all in one place, using real code, in your favorite language.
+
+Pulumi's model shares a lot with these systems. Although Pulumi programs are written in imperative, familiar languages,
```

Comment:
This paragraph doesn't make sense wedged in where it is. It interrupts the flow between the second and fourth paragraphs, and is essentially saying the same thing as the fifth. We should reposition it, make it work, or remove it.

Reply:
Since it is redundant I opted to remove it. No reason to have redundant content IMO.